database-plugins/src/main/java/co/cask/hydrator/plugin/DBUtils.java

```diff
@@ -58,6 +60,43 @@ public static void cleanup(Class<? extends Driver> driverClass) {
   }
 
   /**
+   * Ensures that the JDBC Driver specified in configuration is available and can be loaded. Also registers it with
+   * {@link DriverManager} if it is not already registered.
+   */
+  public static JDBCDriverShim ensureJDBCDriverIsAvailable(Class<? extends Driver> jdbcDriverClass,
```

Comment:
`@Nullable` since this method can return null. Also good to have it in Javadoc saying under what condition it will return `null`.